packages/data/addon/services/cardstack-data.js

```diff
@@ -138,6 +138,16 @@ export default Service.extend({
     }
   },
 
+  async branches() {
+    let url = `${hubURL}/api/branches`;
+    let response = await fetch(url, {
+      headers: this._headers()
+    });
+    let { data } = await response.json();
+
+    return data.map(branch => branch.id);
+  },
```

Comment:
Why is this not simply

```js
this.store.findAll('branch');
```

?

Reply:
that's a good point, if you emit schema via the static-models, the hub framework will codegen an ember data `branch` model for you.

Reply:
I've updated this now to use the store directly 👍

Reply:
Is the `return branches.map(branch => branch.id);` still necessary then?

Reply:
Yes, I think we should simplify this to just return the list of branch models.